storybook/shared/wrapStory.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+import * as React from 'react'
+import { Dev } from 'reflex'
```

Comment:
It should import `Dev` directly from `reflex/src/dev/dev.js` to ensure the production build also shows the overlay